app/src/main/res/values/strings.xml

```diff
@@ -1496,6 +1496,7 @@
     <string name="__do_you_want_to_change_settings_or_start_sensor">\n\nDo you want to change settings or start sensor?</string>
     <string name="change_settings">Change settings</string>
     <string name="do_you_want_to_delete_and_reset_the_calibrations_for_this_sensor">Do you want to delete and reset the calibrations for this sensor?</string>
+    <string name="sensor_stop_confirm">Do you want to Stop sensor and end the current session?</string>
```

Comment:
Please change it into "Do you want to stop this sensor." IMHO, many users do not understand what "session" means. The term in not introduced, AFAIK.

Reply:
Done